apps/Discord/install-32

```diff
@@ -21,3 +21,4 @@ rm -f ~/.local/share/applications/discord.desktop 2> /dev/null
 echo "downloading discord-electron..."
 download || error "Failed to download latest release!"
 sudo apt -f -y install ./electron-discord-webapp*armhf.deb
```

Comment:
AFAIK `apt` **has issues with wildcards**.
However, it [supports *patterns*](https://manpages.debian.org/apt/apt-patterns.7).

If you don't want to use regex nor patterns, you could run `wget` with `-O <output-name>.deb` parameters to download and rename a file to `<output-name>`.

Also, you could use `apt-get` instead, as it supports wildcards. It also skips prompting the user to confirm if he really want to install this package (so there's no need to use `-y` flag.

Reply:
I used dpkg -i to install the deb (it supports wildcards) nad then `sudo apt -f install` to fix any missing dependencies.